### DISCUSSION TITLE
IT WORKS - THANK YOU

### DISCUSSION THREAD

**Discussion by acortelyou** [p10tyr/rtsp-to-onvif#8] [General]
I have been trying to get Unifi Protect to work with our existing legacy RTSP cameras that would be difficult to replace for YEARS but previous attempts were never stable (or rather a Pi3 couldn't manage to keep 5 streams going using unifi-cam-proxy).

This worked on first try and with very reasonable CPU usage!

Talk about Christmas coming early, huge surprise that Unifi opened the door and thanks to this project for getting these ancient but still functional IP cameras across the threshold.

THANK YOU

**Comment by jorgelamo** [comment]
You mentioned the pi hardware what are you using? If I'm using the Pi just for this purpose, which version would you recommend? I'll be using it to integrate 14 cameras into my unifi-protect system. They are all 1080 TVI hooked up to an encoder. As they fail, I'll replace them with a unified camera.
